cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -250,11 +250,10 @@ features:
         totalRuns: Total runs
       DetailView:
         creation: Creation date and time
-        disableConfirmation: Are you sure you want to disable the profile *_{profile}_*? The pipelines currently using this profile will start using the *_native_* profile.
+        disableConfirmation: Are you sure you want to disable the profile *_{profile}_*? The pipelines currently using this profile will fail to start.
```

Comment:
should not use `pipelines` here, since this will apply beyond pipelines

Reply:
@bdmogal @leacd Any comment on the message here?

Reply:
Any programs or pipelines using this profile...